Singularity.v3.9-torch1.10.2-dj0.12.7.def

```diff
@@ -0,0 +1,77 @@
+BootStrap: docker
+From: ubuntu:21.10
+
+%labels
+MAINTAINER Christoph Blessing <chris24.blessing@gmail.com>
+
+%post
+    apt update && apt install -y software-properties-common
+    add-apt-repository universe
+
+
+
+    apt update && apt install -y \
+        build-essential \
+        git \
+        wget \
+        vim \
+        curl \
+        zip \
+        zlib1g-dev \
+        unzip \
+        pkg-config \
+        libblas-dev \
+        liblapack-dev \
+        python3-tk \
+        python3-wheel \
+        graphviz \
+        libhdf5-dev \
+        python3.9 \
+        python3.9-dev \
+        python3.9-distutils \
+        python3-testresources \
+        swig
+    apt-get clean
+
+    ln -s /usr/bin/python3.9 /usr/local/bin/python
+    ln -s /usr/bin/python3.9 /usr/local/bin/python3
+
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    python3.9 get-pip.py
+    rm get-pip.py
+    python3.9 -m pip --no-cache-dir install --upgrade pip
+
+    python3.9 -m pip --no-cache-dir install \
+        blackcellmagic\
+        pytest \
+        pytest-cov \
+        numpy \
+        matplotlib \
+        scipy \
+        pandas \
+        jupyter \
+        scikit-learn \
+        scikit-image \
+        seaborn \
+        graphviz \
+        gpustat \
+        h5py \
+        gitpython \
+        Pillow==8.0.1 \
+        jupyterlab \
+        datajoint==0.12.7
+    
+    python3.9 -m pip --no-cache-dir  install \
+        torch==1.10.2+cu113 \
+        torchvision==0.11.3+cu113 \
+        torchaudio==0.10.2+cu113 \
+        -f https://download.pytorch.org/whl/cu113/torch_stable.html
+
+%environment
+    export SHELL=/bin/sh
+
+%runscript
+    exec /bin/sh -c "$@"
+
+%startscript
+    jupyter lab
```

Comment:
Please add a newline to the end of the file.

Reply:
Shouldn't we stay on 0.12.7 as indicated in the name of the .def file? (I'd like to avoid changing the dj version in this file, too, since I'm sure that 0.12.7 works for me, whereas I'm unsure about 0.12.9 - if I remember correctly there were some issues when I tried that during debugging dj issues in last summer)

Will add a new line and push